libexec/bats-core/bats

```diff
@@ -363,13 +464,23 @@ for filename in "${arguments[@]}"; do
 
   if [[ -d "$filename" ]]; then
     shopt -s nullglob
-    if [[ "$recursive" -eq 1 ]]; then
+    if [[ "$recursive" == "0" ]]; then
```

Comment:
Why?

Reply:
i ask myself as well. it's a bad leftover of a small refactoring.
just wanted to make the global var **_recursive_** acting conform to shell standards. 0=true; other false
did not like that 0 is true if it is in recursive mode.

my preferred way would be:
recursive=1  # set globally to false

and the commented if written as
if  [[ "$recursive" -eq 0]]; then